doc/kube-setup-data-ingestion-job.md

```diff
@@ -0,0 +1,71 @@
+The data-ingestion-job is specific to DataSTAGE.
+
+To prep:
+
+Specify an image for the pipeline in the `versions` block of your manifest, for example:
+`"data-ingestion-pipeline": "quay.io/cdis/datastage-data-ingestion"`
+
+Fill out the config file (see below for example format) with creds here: `$(gen3_secrets_folder)/g3auto/data-ingestion-job/data_ingestion_job_config.json`
+
+Place a newline-separated list of phs IDs here: `$(gen3_secrets_folder)/g3auto/data-ingestion-job/phsids.txt`
+
+Optionally place a "data\_requiring\_manual\_review.tsv" file here: `$(gen3_secrets_folder)/g3auto/data-ingestion-job/data_requiring_manual_review.tsv`
+
+If `CREATE_GENOME_MANIFEST` is false, the genome file manifest is required to live in `g3auto/data-ingestion-job/`. Optionally place a "genome\_file\_manifest" here: `$(gen3_secrets_folder)/g3auto/data-ingestion-job/genome_file_manifest.csv`
```

Comment:
```suggestion
If `CREATE_GENOME_MANIFEST` is false, the genome file manifest is required to live in a "genome\_file\_manifest" here: `$(gen3_secrets_folder)/g3auto/data-ingestion-job/genome_file_manifest.csv`
```

Is this true? it seems like the "optionally" is actually not optional if this is false